src/Symfony/Component/Serializer/Normalizer/RouteContextNormalizer.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Normalizer;
+
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
```

Comment:
This will require to add the `symfony/router` into the `composer.json` for Serializer component.